### PR TITLE
Release 132

### PR DESCRIPTION
**Pull request by dperpeet**

### CODE REVIEW

File: content/cockpit-132.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JIHQmFHOrO4?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### Show session virtual Machines on the machines page
+
+Libvirt differentiates between system virtual machines and session ones, which are tied to the user. In Cockpit
+all the virtual machines accessible to the logged in user, system and session, are now shown in a combined list.
+Also support has been added for external providers other than libvirt.
+
+![Display Language Dialog](http://cockpit-project.org/blog/images/cockpit-machines-session.png)
```
Comment (stefwalter) [inline content/cockpit-132.md]: The Alt text here should be changed.

File: content/cockpit-132.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+NetworkManager version 1.4. For bonds, the MAC addresses are shown and can be edited starting with NetworkManager
+version 1.6. Take a look at the video below for a demo.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JIHQmFHOrO4?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### Show session virtual Machines on the machines page
+
+Libvirt differentiates between system virtual machines and session ones, which are tied to the user. In Cockpit
+all the virtual machines accessible to the logged in user, system and session, are now shown in a combined list.
+Also support has been added for external providers other than libvirt.
```
Comment (stefwalter) [inline content/cockpit-132.md]: No other provider is present. This is an implementation detail at this point. If you're going to change anything else, I'd suggest taking out this last line. The first two lines here make sense.

File: content/cockpit-132.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+![Display Language Dialog](http://cockpit-project.org/blog/images/cockpit-machines-session.png)
+
+### Cockpit file API improved
+
+The Cockpit [javascript API for file interaction](http://cockpit-project.org/guide/132/cockpit-file.html) has been
+extended to enable reading of non-memory-mappable files such as in ```/proc```. Also, there is now an option to
+specify the ```max_read_size``` which will make sure that the read fails if the file is too large.
```
Comment (stefwalter) [inline content/cockpit-132.md]: Not user or admin or deployment facing. Not sure this needs to be included.

Comment (stefwalter) [inline content/cockpit-132.md]: In other words, it's a bug that we didn't already have this in place.

File: content/cockpit-132.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+### The subpackage cockpit-test-assets has been renamed to cockpit-tests
+
+We want to have installable tests and renaming the subpackage is a step on the way to achieving that. This is the
+package for installable unit tests and integration tests. At some point, this subpackage will then also have the
+right dependencies to actually run the tests instead of maintaining a document in the git repository.
```
Comment (stefwalter) [inline content/cockpit-132.md]: This doesn't seem ready. I wouldn't include this. In addition on most (Fedora, CentOS, Debian) operating systems we didn't build cockpit-test-assets until now.

File: content/cockpit-132.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+### Packaging fixes on Debian/Ubuntu
+
+Packaging is now improved on Debian/Ubuntu. Also, dependencies on cockpit-bridge are relaxed to match how Cockpit
+behaves on rpm based distributions.
```
Comment (stefwalter) [inline content/cockpit-132.md]: Unless this fixes a real user facing bug, I wouldn't include it.

File: content/cockpit-132.md (hunk 2 above)
Comment (dperpeet) [inline content/cockpit-132.md]: done

File: content/cockpit-132.md (hunk 1 above)
Comment (dperpeet) [inline content/cockpit-132.md]: done

File: content/cockpit-132.md (hunk 5 above)
Comment (dperpeet) [inline content/cockpit-132.md]: removed

File: content/cockpit-132.md (hunk 4 above)
Comment (dperpeet) [inline content/cockpit-132.md]: removed

File: content/cockpit-132.md (hunk 3 above)
Comment (dperpeet) [inline content/cockpit-132.md]: removed

**Comment by dperpeet** [comment]
updated

**Pull request merged**